crates/dbsp/src/operator/dynamic/consolidate.rs

```diff
@@ -5,14 +5,15 @@ use std::{borrow::Cow, marker::PhantomData};
 
 use crate::{
     circuit::{
+        circuit_builder::StreamId,
         operator_traits::{Operator, UnaryOperator},
-        Circuit, GlobalNodeId, OwnershipPreference, Scope, Stream,
+        Circuit, OwnershipPreference, Scope, Stream,
     },
     circuit_cache_key,
     trace::{Batch, BatchReader, Trace},
 };
 
-circuit_cache_key!(ConsolidateId<C, D>(GlobalNodeId => Stream<C, D>));
+circuit_cache_key!(ConsolidateId<C, D>(StreamId => Stream<C, D>));
```

Comment:
I assume you have searched and replaced all of these

Reply:
I did